### ISSUE TITLE
No audio in Windows XP Tour segments

### ISSUE THREAD

**Issue by torokati44** [ruffle-rs/ruffle#5096]
As reported by user Silica on Discord.

After opening `main.swf`, then clicking "Skip Intro", then selecting the first ("Best for Business") section, the expected audio is not playing.

This is what I get in the debug output:
```
[2021-08-19T17:34:37Z WARN  ruffle_core::avm1::globals::sound] Sound.attachSound: Sound 'TOUR16' not found -- in [Actions Parent] / [Frame] / playVO(string, number)
[2021-08-19T17:34:37Z WARN  ruffle_core::avm1::globals::sound] Sound.start: No sound is attached -- in [Actions Parent] / [Frame] / playVO(string, number)
```

I have a suspicion that this fallback mechanism might be to blame, because the tour is made up of several SWF files, but couldn't figure out what it should do instead (and I don't have access to any Adobe software for experimentation):
https://github.com/ruffle-rs/ruffle/blob/58c907e98576b2bf70423c7b083f0575d7dcbf5b/core/src/avm1/globals/sound.rs#L73-L84

It's embedded here: https://famitama.xyz/xp/index.html
The entire set of files can be downloaded at once from: https://famitama.xyz/xp/mmTour.7z
Originally acquired from: https://archive.org/details/WinXPTour

Please note that on case-sensitive filesystems (practically: on Linux) you have to rename some of the files to look like this (only the capitalization changes):

```intro.swf
intro.txt
main.swf
nav.swf
nav.txt
segment1.swf
Segment1.txt
segment2.swf
Segment2.txt
segment3.swf
segment3.txt
segment4.swf
segment4.txt
segment5.swf
segment5.txt
```

**Comment by parkerlreed** [comment]
I downloaded the archive.org but there is no main.swf. Where is this coming from?

**Comment by torokati44** [comment]
Try the other link, the .7z archive, that has them all extracted from the .exe, sorry for not being entirely clear!

**Comment by parkerlreed** [comment]
Ok thanks. The 7z link is broken when trying to download in Chrome (never does anything on clicking/open in new tab). Had to download from the command-line.

**Comment by torokati44** [comment]
Oh, it must be because it's an `http://` URL... 🤦‍♂️
It's not the best UX if you ask me... https://support.google.com/chrome/thread/77421860/clicking-download-link-does-nothing-at-all?hl=en
Let me fix it quick!

**Comment by dginovker** [comment]
Audio works on 0.1.0.544

**Comment by p0008874** [comment]
I think this issue was already fixed.

**Comment by n0samu** [comment]
Seems that this was fixed by #6352
